### PR TITLE
Fix centering current index

### LINKED ISSUE #615: Centering of current index row is wrong on the first index toggle
Toggling index chooses index row corresponding to the current page, and centers it in the index view.

However, on the first opening of the index, the current index row is top aligned (it is necesarry to have long enough index to notice such behavior). Afterwards, the centering works fine.

I have no idea why this happens, any pointers would be welcome.

### PR DESCRIPTION
**Pull request by laomuon**
Fixes: #615 by making the scrolling to be execute after everything else

### CODE REVIEW

**Comment by jdujava** [comment]
Seems to work alright, thanks for looking into this!

File: zathura/utils.c [hunk 1]
```diff
@@ -134,9 +141,11 @@ void index_scroll_to_current_page(zathura_t* zathura) {
   gtk_tree_model_foreach(model, search_current_index, &search_data);
 
   GtkTreePath* current_path = gtk_tree_model_get_path(model, &search_data.current_iter);
+  zathura->global.current_index_path = gtk_tree_path_to_string(current_path);
```
Comment (sebastinas) [inline zathura/utils.c:144]: Does this need to be freed at some point?

Comment (laomuon) [reply zathura/utils.c:144]: I assume here that you want for it to be freed each time a new value is assigned

Comment (sebastinas) [reply zathura/utils.c:144]: Yes, and when cleaning up.

File: zathura/zathura.h [hunk 2]
```diff
@@ -3,6 +3,7 @@
 #ifndef ZATHURA_H
 #define ZATHURA_H
 
+#include "glib.h"
```
Comment (sebastinas) [inline zathura/zathura.h:6]: This is not required here.

**Pull request merged**